### ISSUE TITLE
Error Message: REF_NOT_FOUND When Running 'hugo server' Locally

### ISSUE THREAD

**Issue by DevyanshuNegi** [CollaboraOnline/CollaboraOnline.github.io#111]
on running " hugo server " in local machine.
I am getting this error message  REF_NOT_FOUND

It is because some path variable must be wrong or does not exist.

    I tried it on two different machines both in ubuntu and windows and getting the same error.
![Screenshot from 2024-03-01 15-41-36](https://github.com/CollaboraOnline/CollaboraOnline.github.io/assets/100033611/a542550b-9821-4395-bca7-f56a21a2a3f8)

**Comment by Minion3665** [comment]
That's interesting, it sounds like the same issue is coming up in Windows and Ubuntu, so for now let's see if we can diagnose the problem in Ubuntu, as I'm more familiar with that...

The error seems to point to some missing files in `/content/snippets` and `/content/post`... so that we can pinpoint the issue, do you mind letting us what the output of

```
ls content/snippets/ content/post/
```

is?

for me, it shows
```
[skyler@canvasback co-build-docs]$ ls content/snippets/ content/post/
content/snippets/:
ash-vimrc.md  darshan-launch.md  gokay-launch.md  pranam-c-cpp-properties.md  pranam-launch.md  pranam-tasks.md

content/post/:
_index.md  build-code-android.md  build-code-ios.md  build-code.md  communicate.md  debug-code.md  easyhacks.md  faq.md  filebugs.md  ide.md  translate.md
```

Also, so that we can more effectively reproduce this ourselves, do you mind telling us the version of hugo you're using? If you run `hugo` it'll give you some output which tells you
```
[skyler@canvasback co-build-docs]$ hugo
Start building sites … 
hugo v0.117.0+extended linux/amd64 BuildDate=unknown VendorInfo=nixpkgs


                   | EN   
-------------------+------
  Pages            | 103  
  Paginator pages  |   0  
  Non-page files   |  18  
  Static files     |  29  
  Processed images |  24  
  Aliases          |  15  
  Sitemaps         |   1  
  Cleaned          |   0  

Total in 161 ms
```

**Comment by DevyanshuNegi** [comment]
this is the hugo version im my case
hugo v0.123.3-a75a659f6fc0cb3a52b2b2ba666a81f79a459376+extended linux/amd64 BuildDate=2024-02-23T17:09:20Z VendorInfo=brew

and in my case if you replace the file path from
"/content/snippets/darshan-launch.md"
to
"/snippets/darshan-launch.md"
for all six paths then it works.

**Comment by Minion3665** [comment]
Thank you for your help in debugging and thanks for your fix! After upgrading to hugo v0.123.6+extended I also see this issue, presumably this stopped being supported in one of the versions and we didn't notice. I'm going to make a note of this on your commit fixing this, and then merge it
